Restore.cmd

```diff
@@ -41,8 +41,8 @@ call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\dev14.project.json"
 echo Restoring packages: Toolsets (Dev15 VS SDK build tools)
 call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\dev15.project.json" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
 
-echo Restoring packages: Toolsets (Dev15 VS SDK 'Willow' build tools)
-call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\dev15Willow.project.json" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
+echo Restoring packages: Toolsets (Dev15 VS SDK Preview 5build tools)
+call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\dev15p5.project.json" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
```

Comment:
Any reason to keep this (and the one above that is for P3) around if we're only ever going to build with Dev14 or Dev15 RC?